phoenix-pherf/src/main/java/org/apache/phoenix/pherf/workload/MultiThreadedDifferTest.java

```diff
@@ -0,0 +1,51 @@
+package org.apache.phoenix.pherf.workload;
+
+import javafx.util.Pair;
+import org.apache.phoenix.pherf.configuration.Query;
+import org.apache.phoenix.pherf.configuration.Scenario;
+import org.apache.phoenix.pherf.configuration.XMLConfigParser;
+import org.apache.phoenix.pherf.result.DataModelResult;
+import org.apache.phoenix.pherf.result.ThreadTime;
+import org.apache.phoenix.pherf.rules.RulesApplier;
+import org.apache.phoenix.util.EnvironmentEdge;
+import org.apache.phoenix.util.EnvironmentEdgeManager;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.sql.ResultSet;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class MultiThreadedDifferTest {
+
+    private static class MyClock extends EnvironmentEdge {
+
+        public MyClock () {}
+
+        @Override
+        public long currentTime() {
+            return System.currentTimeMillis();
+        }
+    }
+
+    @Test
+    public void testTimeout() throws Exception {
+        MyClock myClock = Mockito.mock(MyClock.class);
+        Mockito.when(myClock.currentTime()).thenReturn(0L,5000L); //currentTime() gets called twice during MTR construction
```

Comment:
You mean it gets called twice when "diffQuery" is called, not in the constructor right? Can you change the comment accodingly?